lib/reek/examiner.rb

```diff
@@ -62,12 +63,10 @@ def smelly?
     private
 
     def run
-      @sources.each do |source|
```

Comment:
Awesome, this has been bugging me everytime I came around this piece of source but I never got to fix it.:)